### DISCUSSION TITLE
photonstorm links in readme are dead

### DISCUSSION THREAD

**Discussion by steinhaug** [pixijs/pixijs#10064] [General]
Looks like the links for [photonstorm's Balls Demo](http://gametest.mobi/pixi/balls) and [photonstorm's Morph Demo](http://gametest.mobi/pixi/morph) are down, gives an "Account suspended" page. This early in the month it cannot be bandwidth reasons  so I'm guessing the problem is permanent.

**Comment by bigtimebuddy** [comment]
Thanks for pointing out. Welcome PR to remove these!
